components/Common/Button/index.module.scss

```diff
@@ -0,0 +1,106 @@
+@import 'styles/new/index';
+
+.button {
+  @apply px-[18px]
+   py-[10px]
+   font-semibold
+   relative;
+
+  &.primary {
+    @apply bg-green-600
+     text-white
+     rounded
+     border
+     border-green-600
+     shadow-sm;
+
+    &:hover {
+      @apply bg-green-700
+       border-green-700;
+    }
+
+    &:focus {
+      @apply bg-green-700
+       border-green-700;
+    }
+
+    &:disabled {
+      @apply bg-green-600
+       opacity-50;
+    }
+  }
+
+  &.secondary {
+    @apply text-neutral-800
+     rounded-lg
+     disabled:opacity-50
+     dark:text-neutral-200/100;
+
+    &:hover {
+      @apply text-neutral-800
+       bg-neutral-100
+       dark:bg-neutral-900
+       dark:text-neutral-200;
+    }
+
+    &:disabled {
+      @apply bg-neutral-100/0;
+    }
+
+    &:focus {
+      @apply text-neutral-800
+       bg-neutral-100
+       dark:bg-neutral-900
+       dark:text-neutral-200;
+    }
+  }
+
+  &.special {
+    @apply bg-green-600/10
+     text-white
+     shadow-sm
+     rounded-lg
+     border
+     border-green-600/30;
+
+    &::before {
+      @apply absolute
+       h-full
+       w-full
+       top-0
+       right-0;
+      @include glow-backdrop(8em, #5fa04e);
```

Comment:
Same here, if we could use one of our color variables? Not sure if it works tho

Reply:
It doesn't work here, we could create a new variables files in which we can re-declare tailwind color tokens in sass variables

Reply:
That's not needed, you can invoke tailwind theme colours directly from CSS https://stackoverflow.com/a/67281604

Reply:
@ovflowd This doesn't seems to work with sass https://tailwindcss.com/docs/using-with-preprocessors#using-sass-less-or-stylus

Reply:
You need to use @theme instead of just theme when using with scss

Reply:
It should work, if you don't combine with sass functions. Because sass gets compiled first, so the alternative here is to not use mixins?

Let's try to see in another perspective, why can't we use tailwinds gradient generator for the gradients that we need?

Reply:
Maybe another approach is to not use Sass at all and only use PostCSS as otherwise we are using two preprocessors at once.. 

I will think about this

Reply:
Because tailwind doesn't support radial gradient, and I am using radial gradient

Reply:
It does not, but you can add radial-gradients on Tailwind config by yourself https://github.com/tailwindlabs/tailwindcss/discussions/2599#discussioncomment-102667

Reply:
This exact one should work: https://play.tailwindcss.com/kQvzcR8OF8 (check the config tab)

Reply:
We can add radial gradients this way but I can't figure out how to replicate the exact style, it seems it is different from what I used, I think it lacks control tbh.

Reply:
That's fine, let's keep it the way it is.